traffic-mirroring/mirroring.c

```diff
@@ -959,13 +958,11 @@ int main(int argc, char **argv)
                             "address %s",
                             dst_addr[i].addr);
                     ret = EXIT_FAILURE;
-                    free(dst_key);
                 }
```

Comment:
Here we will end-up updating the Map eventhough this Conversion fails. Please check this error case if we need to return here before updating the Map.